process/README.md

```diff
@@ -3,40 +3,60 @@
 v1.6, previously "project proposal process v1.5"
 
 ## Introduction 
```

Comment:
Given this file is fairly long, we should consider adding header links to commonly sought information. Like "How to move levels" "timelines"

Reply:
Added table of contents